pages/common/phploc.md

```diff
@@ -0,0 +1,31 @@
+# phploc
+
+> A tool for quickly measuring the size and analyzing the structure of a PHP project.
+
+- Analyse a directory and print the result:
+
+`phploc {{path/to/directory}}`
+
+- Include only specific files from a comma-separated list (globs are allowed):
+
+`phploc {{path/to/directory}} --names {{files}}`
+
+- Exclude specific files from a comma-separated list (globs are allowed):
+
+`phploc {{path/to/directory}} --names-exclude {{files}}`
+
+- Exclude a specific directory from analysis:
+
+`phploc {{path/to/directory}} --exclude {{path/to/directory}}`
```

Comment:
I think it might be helpful if you change the exclude directory to `path/to/exclude_directory` to differentiate.